packages/vaadin-notification/src/vaadin-notification.js

```diff
@@ -457,7 +453,7 @@ class NotificationElement extends ThemePropertyMixin(ElementMixin(PolymerElement
     }
     this._instance = new this._notificationTemplate._Templatizer({});
 
-    this._card = this.$['vaadin-notification-card'];
+    this._card = this.shadowRoot.querySelector('vaadin-notification-card');
```

Comment:
... here we don't need to use `this._card || ...` because it is called only once before the notification is moved to the notification container.